### PR TITLE
feat(samples): new sample application (SG Hugging Face)

### PR DESCRIPTION
**Pull request by krokoko**
*Issue #, if available:*

*Description of changes:*

Add a new sample: this PR provides a sample application which deploys a SageMaker real-time endpoint hosting a mistral 7B model from Hugging Face, and an AWS Lambda function to run inference requests against that endpoint.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by dineshSajwan — DISMISSED** [review]

File: samples/sagemaker_huggingface_model/test/sagemaker_huggingface_model.test.ts
Comment (MichaelWalker-git) [inline samples/sagemaker_huggingface_model/test/sagemaker_huggingface_model.test.ts:1]: Nit: Do we need this file?

**Review by MichaelWalker-git — APPROVED** [review]
Overall looks good. Small comment

**Review by scottschreckengaust — DISMISSED** [review]

File: samples/sagemaker_huggingface_model/test/sagemaker_huggingface_model.test.ts
Comment (krokoko) [reply samples/sagemaker_huggingface_model/test/sagemaker_huggingface_model.test.ts:1]: nope, will remove it. Auto generated by cdk init, sorry about that :)

Comment (krokoko) [reply samples/sagemaker_huggingface_model/test/sagemaker_huggingface_model.test.ts:1]: Done in [d595c9c](https://github.com/aws-samples/generative-ai-cdk-constructs-samples/pull/17/commits/d595c9c014465515c04acba92db54ff4e14bcec9)

**Review by scottschreckengaust — APPROVED** [review]

**Pull request merged**